spec/api/v1/schemas.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require './spec/api/v1/schemas/types'
+require './spec/api/v1/schemas/errors'
+require './spec/api/v1/schemas/metadata'
+require './spec/api/v1/schemas/hosts'
+require './spec/api/v1/schemas/benchmarks'
+require './spec/api/v1/schemas/profiles'
+require './spec/api/v1/schemas/rule_results'
+require './spec/api/v1/schemas/rules'
+
+module Api
+  module V1
+    module Schemas
+      include Types
+      include Errors
+      include Metadata
+      include Hosts
+      include Benchmarks
+      include Profiles
+      include RuleResults
+      include Rules
+
+      SCHEMAS = {
```

Comment:
I almost used methods instead of constants, but none of this content is dynamic. We can still call methods from constants, such as `ref_schema('foo')`, as seen in the types schemas. Furthermore, rubocop's method length cop does not apply to constants, and some of these specs are still quite long, well past the method length limit.